Changelog.md

```diff
@@ -10,6 +10,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add `VK_KHR_present_wait` extension (#493)
 
+### Changed
+
+- Removed `instance` function from `extensions::ext::ExtendedDynamicState` and `extensions::khr::PushDescriptor` (#494)
```

Comment:
Should we mention that these were replaced with `device` instead?

Also, theoretically we should list the removal of the `instance` and `device` parameters from the functions in the last two commits.

Reply:
I'll cleanup the Changelog.

Reply:
Fyi https://keepachangelog.com/en/1.0.0/ says to have a special `### Removed` header for these :)

(Though in the most cases we're just moving arguments because they're better, or already, provided elsewhere)

Reply:
Yes, and I can't find a good wording (precise and succinct).

Reply:
Giving it another try...

Reply:
Not saying it any different than it is may be a good start. For example, nobody really knows what "normalize" means in this context. Just stating that `Extension X and Y are now exposing `fn device()` instead of `fn instance()` because they are device extensions` seems simple and good enough to me?